tests/components/config/test_config_entries.py

```diff
@@ -216,7 +216,9 @@ def async_supports_options_flow(cls, config_entry):
 
 async def test_remove_entry(hass: HomeAssistant, client) -> None:
     """Test removing an entry via the API."""
-    entry = MockConfigEntry(domain="demo", state=core_ce.ConfigEntryState.LOADED)
+    entry = MockConfigEntry(
+        domain="kitchen_sink", state=core_ce.ConfigEntryState.LOADED
+    )
```

Comment:
Now that `demo` can be unloaded, the `config` tests were failing.
I think it makes more sense to use the `kitchen_sink` integration here...

Reply:
Seems okey